### PR TITLE
Task/create login page

### PR DESCRIPTION
**Pull request by ghassan-hejazi**
create login  page
add package google font

### CODE REVIEW

File: lib/features/home/presentation/pages/home_page.dart [hunk 1]
```diff
@@ -28,6 +28,11 @@ class HomePage extends GetView<Locale>  {
                 ElevatedButton(
                     onPressed: () => myLocaleController.changeLanguage('ar'),
                     child: Text('arbic'.tr)),
+                    ElevatedButton(
```
Comment (AbdalazizAbdallah) [inline lib/features/home/presentation/pages/home_page.dart:31]: Spelling Wrong please correct this..
arabic 😉

File: lib/features/home/presentation/pages/home_page.dart [hunk 2]
```diff
@@ -28,6 +28,11 @@ class HomePage extends GetView<Locale>  {
...
+                    onPressed: () {
+                      Get.toNamed('/loginpage');
```
Comment (AbdalazizAbdallah) [inline lib/features/home/presentation/pages/home_page.dart:33]: Use App Route

File: lib/features/home/presentation/pages/home_page.dart [hunk 3]
```diff
@@ -28,6 +28,11 @@ class HomePage extends GetView<Locale>  {
...
+                    },
+                    child: const Text('login')),
```
Comment (AbdalazizAbdallah) [inline lib/features/home/presentation/pages/home_page.dart:35]: Use translate file as 'login'.tr

File: lib/features/home/presentation/pages/login_page.dart [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+class LoginPage extends GetView<LoginController> {
+  LoginPage({Key? key}) : super(key: key);
+  bool va1 = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: ListView(children: [
+        Container(
+          padding: const EdgeInsets.only(top: 51),
```
Comment (AbdalazizAbdallah) [inline lib/features/home/presentation/pages/login_page.dart:17]: Use screen utils as 51.r

File: lib/features/home/presentation/pages/login_page.dart [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+          alignment: Alignment.topCenter,
+          width: double.infinity,
+          height: 122,
+          child: Text(
+            'daddyEat',
```
Comment (AbdalazizAbdallah) [inline lib/features/home/presentation/pages/login_page.dart:22]: Translate file

File: lib/features/home/presentation/pages/login_page.dart [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+            ),
+          ),
+          color: const Color(0xFF2A2A2A),
+        ),
+        const SizedBox(height: 39),
+        Text(
+          'Are you ready to go?',
+          textAlign: TextAlign.center,
+          style: GoogleFonts.openSans(
+            color: const Color(0xFF2A2A2A),
```
Comment (AbdalazizAbdallah) [inline lib/features/home/presentation/pages/login_page.dart:36]: Put this in Color App and use bootstrap scheme primary and warning etc

File: lib/features/home/presentation/pages/login_page.dart [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+            fontSize: 24,
+            fontWeight: FontWeight.w600,
+          ),
+        ),
+        const SizedBox(height: 16),
+        Text(
+          'signin now',
+          textAlign: TextAlign.center,
+          style: GoogleFonts.openSans(
```
Comment (AbdalazizAbdallah) [inline lib/features/home/presentation/pages/login_page.dart:45]: Move this Into Style App file

**Comment by AhmedGhayada23** [comment]
resolve conflicts and Create inside the features auth folders please

File: lib/core/localization/ar.dart [hunk 8]
```diff
@@ -2,4 +2,15 @@ Map<String, String> ar = {
   'home_page': 'الصفحة الرئيسية',
   'Select_language': 'اختر اللغة',
   'arbic': 'عربي',
+  'login': 'تسجيل دخول',
+  'daddyEat': 'أبي',
```
Comment (fahjan) [inline lib/core/localization/ar.dart:6]: ههههههه
change it to daddyEat

File: lib/core/localization/ar.dart [hunk 9]
```diff
@@ -2,4 +2,15 @@ Map<String, String> ar = {
...
+  'ready_to_go': 'هل أنت مستعد للذهاب؟',
+  'signin_now': 'قم بتسجيل دخولك الآن',
+  'iD_Number': 'ID Number',
+  'password': 'كلمة المرور',
+  'remember_me': 'تذكرني',
```
Comment (fahjan) [inline lib/core/localization/ar.dart:11]: تذكر بيانات الدخول

File: lib/core/localization/ar.dart [hunk 10]
```diff
@@ -2,4 +2,15 @@ Map<String, String> ar = {
...
+  'Login': 'الدخول',
+  'signin': 'تسجيل الدخول',
+  'forgot_password':'هل نسيت كلمة السر؟',
```
Comment (fahjan) [inline lib/core/localization/ar.dart:14]: 'password': 'كلمة المرور',
change السر الر المرور

File: lib/features/home/presentation/pages/login_page.dart [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+                    borderRadius: BorderRadius.circular(AppSizes.r24),
+                    borderSide: BorderSide(
+                      color: AppColors.grey,
+                      width: AppSizes.r2,
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(height: AppSizes.r12),
+              TextFormField(
```
Comment (fahjan) [inline lib/features/home/presentation/pages/login_page.dart:96]: extract this to custom widget

File: lib/utils/routing/app_routes.dart [hunk 12]
```diff
@@ -1,3 +1,5 @@
 class AppRoutes {
-  static const String home = '/';
+  static const String home = '/home';
+  static  String loginpage = '/loginpage';
```
Comment (fahjan) [inline lib/utils/routing/app_routes.dart:3]: use camel case style please: loginPage

**Comment by fahjan** [comment]
and resolve conflicts

**Review by mohammedalian — APPROVED** [review]
Good Job ^_^

File: lib/features/login_screen/presentation/pages/login_page.dart [hunk 13]
```diff
@@ -0,0 +1,143 @@
+// ignore_for_file: sort_child_properties_last, must_be_immutable, prefer_const_constructors_in_immutables
```
Comment (fahjan) [inline lib/features/login_screen/presentation/pages/login_page.dart:1]: use: `const` before widgets need it.

File: lib/features/login_screen/presentation/widgets/text_form_field_widget.dart [hunk 14]
```diff
@@ -0,0 +1,58 @@
+// ignore_for_file: must_be_immutable, non_constant_identifier_names
```
Comment (fahjan) [inline lib/features/login_screen/presentation/widgets/text_form_field_widget.dart:1]: use: const before widgets need it.

**Pull request merged**